### DISCUSSION TITLE
Welcome to DELL-3421-BigSur Discussions!

### DISCUSSION THREAD

**Discussion by alkindivv** [ALKindi-Wilagabrata/DELL-3421#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by L888888888** [comment]
What is the decompression password of the EFI file?

**Reply by alkindivv** [reply]
Its my client EFI file

**Comment by raphaelsdiaz** [comment]
I also need the password for the EFI file, please! What's that?

**Reply by alkindivv** [reply]
Its my client EFI file
